library/src/main/java/com/novoda/downloadmanager/lib/Downloads.java

```diff
@@ -26,8 +26,50 @@
  *
  * @pending
  */
-final class Downloads {
-    private Downloads() {
+public class Downloads {
+
+    private final String authority;
+
+    /**
+     * The content URI for accessing publicly accessible downloads (i.e., it requires no
+     * permissions to access this downloaded file)
+     */
+    public Uri getPublicityAccessibleDownloadsUri() {
```

Comment:
All these `get...` methods should be extracted into a UriProvider class and we will avoid the weird `Downloads` class param all over the project.

Future PR